### PR TITLE
fix(form): improve validation of choice and multiple choice fields

### PR DESCRIPTION
**Pull request by anehx**
- Display the label of the faulty option in the error message
- Don't accept disabled options as valid values in validation
- Disable inputs of disabled options

### CODE REVIEW

File: packages/form/addon/lib/field.js [hunk 1]
```diff
@@ -801,10 +790,20 @@ export default class Field extends Base {
...
+    const label = Array.isArray(this.selected)
+      ? this.selected.find((selected) => selected.slug === value)?.label
+      : this.selected?.label;
+
+    return validate("inclusion", value, {
+      in: (this.options || [])
+        .filter((option) => !option.disabled)
+        .map(({ slug }) => slug),
+      allowBlank,
+      label: label ?? value,
```
Comment (luytena) [inline packages/form/addon/lib/field.js:806]: For what is the `label` option in the `validate` function?

Comment (anehx) [reply packages/form/addon/lib/field.js:806]: It's used in the translation

File: packages/form/translations/fr.yaml [hunk 2]
```diff
@@ -46,7 +46,7 @@ caluma:
       greaterThanOrEqualTo: "La valeur indiquée dans ce champ ne doit pas être inférieure à {gte} "
       lessThanOrEqualTo: "L'entrée dans ce champ ne doit pas être supérieure à {lte} "
       notAnInteger: "Veuillez entrer un nombre entier"
-      inclusion: "'{value}' n'est pas une valeur valide pour ce champ"
+      inclusion: '"{label}" n''est pas une valeur valide pour ce champ'
```
Comment (luytena) [inline packages/form/translations/fr.yaml:49]: Double `'` so that the `n'est` is displayed correctly?

Comment (anehx) [reply packages/form/translations/fr.yaml:49]: Yes, `'` is the escape character in translations: https://formatjs.io/docs/core-concepts/icu-syntax/#quoting--escaping

**Review by luytena — APPROVED** [review]

**Pull request merged**